src/ucp/wireup/select.c

```diff
@@ -261,16 +268,22 @@ ucp_wireup_select_transport(ucp_ep_h ep, const ucp_address_entry_t *address_list
     unsigned addr_index, best_dst_addr_index;
     int reachable;
     int found;
+    size_t best_max_num_eps;
     uint8_t priority, best_score_priority;
 
     found               = 0;
     best_score          = 0.0;
     best_rsc_index      = 0;
     best_dst_addr_index = 0;
     best_score_priority = 0;
+    best_max_num_eps    = 0;
     p                   = tls_info;
     endp                = tls_info + sizeof(tls_info) - 1;
     tls_info[0]         = '\0';
+    tl_bitmap          &= (select_ctx->tl_bitmap & tl_bitmap);
+    show_error          = (select_ctx->show_error && show_error);
```

Comment:
why do we need `tl_bitmap` and `show_error` as a parameter and as a part of selection context?

Reply:
`tl_bitmap` and `show_error` are passed by callers (RMA/AMO/TAG/... lanes). Some of them use those values, e.g. RMA/AMO:
- `show_error` - `0` if AM emulation is allowed, `1` - otherwise
- `tl_bitmap` - adds already used TLs

when `select_ctx`'s `show_error` and `tl_bitmap` are global ones and set when initializing a selection context